From ed14a45c795510f16b9aba3ec153091178e79d67 Mon Sep 17 00:00:00 2001
From: Bruce Ashfield <bruce.ashfield@gmail.com>
Date: Mon, 14 Jul 2025 13:09:06 -0400
Subject: [PATCH 8/8] aufs: drop writepage

The upstream kernel has dropped writepage, and uses writepages directly.

So we drop our registration of writepage, but keep the function around
to be called internally.

Signed-off-by: Bruce Ashfield <bruce.ashfield@gmail.com>
---
 fs/aufs/dynop.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/fs/aufs/dynop.c b/fs/aufs/dynop.c
index 9cf93d0ade3a..1c992fa47a63 100644
--- a/fs/aufs/dynop.c
+++ b/fs/aufs/dynop.c
@@ -174,7 +174,6 @@ static void dy_aop(struct au_dykey *key, const void *h_op,
 
 	AuDbg("%s\n", au_sbtype(h_sb));
 
-	DySetAop(writepage);
 	DySetAopForce(read_folio);	/* force */
 	DySetAop(writepages);
 	DySetAop(dirty_folio);
-- 
2.39.2

